packages/console-client/src/kiota-client/consoleClient.ts

```diff
@@ -46,10 +46,10 @@ export interface ConsoleClient extends BaseRequestBuilder<ConsoleClient> {
  * @param requestAdapter The request adapter to use to execute the requests.
  */
 export function createConsoleClient(requestAdapter: RequestAdapter) {
-    registerDefaultSerializer(JsonSerializationWriterFactory);
-    registerDefaultSerializer(TextSerializationWriterFactory);
-    registerDefaultSerializer(FormSerializationWriterFactory);
-    registerDefaultSerializer(MultipartSerializationWriterFactory);
+    // registerDefaultSerializer(JsonSerializationWriterFactory);
```

Comment:
why these comments?

Reply:
Because of [this](https://github.com/mia-platform/console-sdk/actions/runs/10526721859/job/29168200929), I was trying to understand why the build job is having problems with  this statement. Locally works fine.